ci/docker/cpu-debug.yaml

```diff
@@ -14,10 +14,9 @@ spack:
       variants:
         - '~cuda'
         - '~openmp'
-    hpx:
+    pika:
       variants:
         - 'build_type=Debug'
-        - 'max_cpu_count=128'
```

Comment:
Yep! HPX master also does that now, but the variant can't be removed from the spack package because of older versions... With pika we got a clean slate so it's gone now.

This is the PR that changed it: https://github.com/STEllAR-GROUP/hpx/pull/5663. The CMake variable `HPX/PIKA_WITH_MAX_CPU_COUNT` is still there, it just defaults to empty, i.e. using dynamic bitsets since the bitsets aren't used in any performance-sensitive loops. So you can still keep setting `HPX/PIKA_WITH_MAX_CPU_COUNT` if you feel like it.